GameOfHive/HiveButton.swift

```diff
@@ -59,6 +59,11 @@ class HiveButton: UIButton {
         backgroundView.backgroundColor = UIColor.menuButtonBackgroundColor
         backgroundView.userInteractionEnabled = false
     }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        backgroundView.frame = bounds
+    }
```

Comment:
Waar is dit voor nodig? Dit gebeurd al met constraints door `backgroundView.constrainToView(self)`